contrib/admin/test-pr

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+# test-pr: submit a PR from an external contributor for Gitlab testing
+
+set -e
+
+# Check arguments
+if [[ "${#}" -ne 3 ]] ; then
+    printf >&2 "test-pr: submit a PR from an external contributor for Gitlab testing\n\n"
+    printf >&2 "\tusage: test-pr GH_USER SOURCE_BRANCH DEST_BRANCH\n\n"
+    printf >&2 "\texample: test-pr arostamianfar defaultcaching issues/2586-allow-enabling-caching\n\n"
+    exit 1
+fi
+
+# Parse arguments
+GH_USER="${1}"
+shift
+SOURCE_BRANCH="${1}"
+shift
+DEST_BRANCH="${1}"
+shift
+
+echo "Fetch ${SOURCE_BRANCH} from ${GH_USER}"
+git fetch "git@github.com:${GH_USER}/toil.git" "${SOURCE_BRANCH}"
+echo "Push to Toil ${DEST_BRANCH}"
+git push git@github.com:DataBiosphere/toil.git "FETCH_HEAD:refs/heads/${DEST_BRANCH}"
```

Comment:
I think we could have travis do this?  Travis could be given credentials and when tests run it'll push a new gitlab branch.

Reply:
I don't want PRs automatically pulled into branches unless they're from people we trust, because running on our CI nodes gives them access into our AWS infrastructure. Especially with the autoscaling setup I'm working on, where the CI nodes will have access to start more machines, which they will eventually need in order to run any launch-cluster tests.

Either someone has to manually check the commit, or we trust the person enough that we can just give them access to push to a branch themselves, is my thinking.

Reply:
Alright, that sounds sane.